go/ses/SendMessage/SendMessage.go

```diff
@@ -0,0 +1,107 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[ses.go.send_message]
+package main
+
+// snippet-start:[ses.go.send_message.imports]
+import (
+    "flag"
+    "fmt"
+
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/ses"
+    "github.com/aws/aws-sdk-go/service/ses/sesiface"
+)
+// snippet-end:[ses.go.send_message.imports]
+
+// snippet-start:[ses.go.send_message.consts]
+const (
+    // Subject is the subject line for the email.
```

Comment:
Lines 20, 23, 28, 31: remove ending period to be consistent with other comments in here

Reply:
Fixed